vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ExhibitionLanguage.java

```diff
@@ -0,0 +1,93 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+@Entity
+public class ExhibitionLanguage {
+
+    public ExhibitionLanguage() {
+        super();
+    }
```

Comment:
is this constructor necessary?

Reply:
Yeah. Actually there is a parameterized constructor added.

Reply:
Ah I see, I didn't see it because constructors should be grouped together after the field declarations ;)